rust/src/quic/quic.rs

```diff
@@ -23,7 +23,7 @@ use super::{
 };
 use crate::applayer::{self, *};
 use crate::core::{AppProto, Flow, ALPROTO_FAILED, ALPROTO_UNKNOWN, IPPROTO_UDP, Direction};
-use std::collections::VecDeque;
+use std::{collections::VecDeque};
```

Comment:
Why this change ?

Reply:
That must have been done by some automated source formatter; I didn't do that myself. Doesn't make sense, really. Happy to remove it.

Reply:
Done